.env.test

```
@@ -19,3 +20,5 @@ FLASK_SKIP_DOTENV=True
 
 TRELLO_BOARD_ID=TEST_TRELLO_BOARD_ID
 
+PRIMARY_CONNECTION_STRING_DB=mongodb://fakemongo.com
+chaostodoSaved=chaostodoSaved
```

Is this meant to be:
```suggestion
USERNAME=chaostodoSaved
```
?

---

Bonus question, is this meant to be DB_NAME rather than USERNAME? Because it looks like this is the name you've assigned to your DB later on regardless of what this value is